src/components/Dialog.tsx

```diff
@@ -327,7 +336,7 @@ export default function Dialog({ color, children, onClose, forceContain }: Dialo
           return (mountPoint?.childElementCount ?? 0) > 1 ? SlideAnimationType.PAGING : SlideAnimationType.CLOSING
       }
     },
-    modal,
+    [fullScreenWrapperRef, modal],
```

Comment:
Would it be safer to use nesting CSS instead of passing an array here?

Reply:
i think this is worth looking into later. i actually tried that approach at first but couldn't get it working. can't spend more time on this atm but i'll circle back.